src/elements/tabs/tab-group/tab-group.ts

```diff
@@ -301,7 +307,14 @@ export class SbbTabGroupElement extends LitElement {
           this._selectedTab = tab;
 
           this._tabContentResizeObserver.observe(tab.relatedContent!);
-          this._selectedTabChanged.emit();
+          this._selectedTabChanged.emit({
+            activeIndex: this._tabs.findIndex((e) => e === tab),
```

Comment:
Probably my original thought was not fully understandable. I thought there were two calls `this._tabs` which do the queryselector stuff. So this additional call could be saved by using const tabs = this._tabs and then in the following usages only as tabs.